### ISSUE TITLE
Request large contients

### ISSUE THREAD

**Issue by jamesol1** [Quidvio/Custom-Floating-Islands#2]
Hi
i was wondering if you'd be able to make a version with very large floating island contients, and then wider air space inbetween. With the floating islands not varying too much in terms of y level. Without floating oceans too ideally, or if there is, only so often and have them be very big.
Thanks!

**Comment by klinbee** [comment]
Sure, I can try that out whenever I get the time. I've got a question though. You mentioned only floating oceans if they are big right? What do you mean by that? If you've seen my recent update to floating islands, in my opinion those floating oceans are always big. Do you just mean like sections where the ocean biome itself is small not to make a floating ocean there?

**Comment by jamesol1** [comment]
yeah they are all big true. Keep them to that size actually. maybe do it without oceans to be fair, they can work as the airspace i suppose. if its possible to add the odd one by all means. sounds like you either get them or dont. and many thanks!!

**Comment by klinbee** [comment]
> yeah they are all big true. Keep them to that size actually. maybe do it without oceans to be fair, they can work as the airspace i suppose. if its possible to add the odd one by all means. sounds like you either get them or dont. and many thanks!!

So you've told me to keep them how they are, and then do it without them, and then to use them as the air space. And then you've said that if it is possible to add the odd one do that. 

I'm very confused as to what you'd actually want since you've said to do 4 different things. Let me give you a list of options and you can select which option(s) you'd like the most, or let me know in more detail what you mean.

1. Leave everything as is.
2. Completely remove the floating oceans in ocean biomes, but add land back to where ocean biomes are.
3. Completely remove the floating oceans in ocean biomes, but leave the land removed. 
4. Make all of the empty space between islands ocean biomes and have floating oceans there. 
5. Leave everything as is, EXCEPT make the large oceans sometimes not generate.
6. Leave everythingn as is, EXCEPT make the large oceans sometimes not generate AND add land where the oceans didn't generate.

**Comment by jamesol1** [comment]
My bad i assumed removing oceans would be the method used for having more air space so i mentioned that. Not fully sure how its all done.
Number 2 would work nicely.

thanks
